### PR TITLE
Plug panel mod for Schurter CD44.1101.151

### PR DESCRIPTION
**Pull request by jatson**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Added a modified plug panel for the Schurter CD44.1101.151 filtered power entry

#### How was it tested? 
The modified panel printed and currently in use on my 2.4

#### Any background context you want to provide?
This is a very reliable power entry. It has filtering and during power off, it will cut the line and neutral as well.

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Comment by MapleLeafMakers** [comment]
image

Parts need to be printable without supports.  these overhangs aren't going to work.

**Comment by jatson** [comment]
@MapleLeafMakers You are absolutely right. I've overlooked this point somehow. Sorry about that.
I've fixed the model. Now it shall be printable w/o support. 
Thanks for the highlight.

**Comment by MapleLeafMakers** [comment]
Looks good to me!

File: printer_mods/README.md [hunk 1]
```diff
@@ -193,6 +193,7 @@ like so:
 | jappaj | [V0.1 FanSaver](./jappaj/FanSaver) | A printable fix for overheating V0.1 blower fans | :heavy_check_mark: | :x: | :x: | :x: | :x:|
 |jasonc |[V2.4 Z-Idler Shoulder Bolt Mod](./jasonc/Z_Idler_Shoulder_Bolt) |Modification to the idler carriers to accept a 25mm shoulder bolt. Ideal for Gates idlers. |:x: |:x: |:heavy_check_mark: |:x:|
 | Jasrags | [Feed slot with filament pass-through](./Jasrags/sw_fee_slot_filament_passthrough) | A feed slot with a filament pass-through | :x: | :x: | :x: | :heavy_check_mark: |:x:|
+| jatson | [Plug panel mod for Schurter CD44.1101.151 power entry](./jatson/plug_panel_schurter) | A modofied plug panel for Voron 2.4 with Schurter CD44.1101.151 power entry modul. The panel has a LAN socket as well. | :x: | :grey_question: | :heavy_check_mark: | :x: |:x:|
```
Comment (Haxk20) [inline printer_mods/README.md:196]: modofied -> modified

File: printer_mods/jatson/plug_panel_schurter/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Plug Panel Mod for Schurter CD44.1101.151
+Connection panel for Voron 2.4 with Schurter CD44.1101.151 power entry modul.
```
Comment (Haxk20) [inline printer_mods/jatson/plug_panel_schurter/README.md:2]: Im guessing you meant module here instead of modul :)

**Review by Haxk20 — CHANGES_REQUESTED** [review]
Other then these 2 typos it looks good :)

File: printer_mods/README.md (hunk 1 above)
Comment (jatson) [reply printer_mods/README.md]: typo fixed

File: printer_mods/jatson/plug_panel_schurter/README.md (hunk 2 above)
Comment (jatson) [reply printer_mods/jatson/plug_panel_schurter/README.md]: typo fixed

**Review by Haxk20 — APPROVED** [review]
Looks good.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Can you just mention somewhere in the readme that this is only compatible with the v2.4r1? r2 uses different skirts that won't work with this model.

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**